app/frontend/components/topNavigation/topNavigationLink.tsx

```diff
@@ -10,9 +10,12 @@ export interface TopNavigationLinkProps {
 
 export const TopNavigationLink = (props: TopNavigationLinkProps): JSX.Element => {
   const router = useRouter()
+  const teamSlug = router.query.teamSlug as string
+
   let classNames = 'mx-3 my-3 bg-transparent  py-1 px-4 font-semibold'
+
   classNames =
-    props.href && router.pathname.startsWith(props.href)
+    props.href && router.pathname.replace('[teamSlug]', teamSlug).startsWith(props.href)
```

Comment:
That's too complicated.
We can use `.asPath` instead of `.pathname` and get rid of the replace function.
![image](https://user-images.githubusercontent.com/1886867/161244513-5bd25d61-4cd6-4f69-ae1d-ee9ba257f22e.png)